src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -14,6 +15,15 @@
 
 namespace Microsoft.CodeAnalysis.FlowAnalysis
 {
+    /// <summary>
+    /// Some basic concepts:
+    /// - Basic blocks are sequences of statements/operations with no branching. The only branching
+    ///   allowed is at the end of the basic block.
+    /// - Regions group blocks together and represent the lifetime of locals and captures, loosely similar to scopes in C#.
+    ///   There are different kinds of regions, <see cref="ControlFlowRegionKind"/>.
+    /// - <see cref="ControlFlowGraphBuilder.SpillEvalStack"/> converts values on the stack into captures.
+    /// - Error scenarios from initial binding need to be handled.
+    /// </summary>
```

Comment:
Thanks for this. It helped me review this PR and I'm sure it will help future readers.